test/fake_modules/bad_es6/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "dependencies": {
+    "optimist": "~0.6.0",
+    "jquery": "^2.1.4"
```

Comment:
It is not a good idea to use real name here, use some special unique names here. If somebody reference a wrong folder, the test will find them.